### ISSUE TITLE
Reinstate "send to expert" option

### ISSUE THREAD

**Issue by kitenetter** [BiologicalRecordsCentre/iRecord#1054]
A couple of verifiers have requested that we reinstate this function:

"In the verifier tab of the iRecord web page, if I raise a query in only have the option to query to the recorder, I used to have a "raise a query with an expert" option as well. If I use the "query" option to forward my question to an expert then the recorder gets a ping as well, but not the email/comment, this has raised a flurry of comments, something I would prefer to avoid ! Is there a work around or will I need to do the "query to an expert" thing outside the system, which will lose the audit trail...."

What scale of job would it be to bring this back?

**Comment by DavidHepper** [comment]
Yes, please reinstate this but without emailing the Recorder. BDS uses experts for migrant species, in particular.

**Comment by MNSmith** [comment]
If you were to reinstate this, please could we have a stored list of potential user defined e-mails to use with this.  Previously it was a bit of a pain to have to go out of iRecord and do a cut and paste of an e-mail address - at theat point it was just as easy to sent an e-mailk directly from ones e-mail system saying "Have a look at Record xxxxxx please.

**Comment by DavidHepper** [comment]
I've been asked again for this, especially so that photos attached to the iR record can easily be shared with the expert.
However, I'm struggling to see how the request by @MNSmith could be implemented without having a flag _per group_ in the user's record indicating expert status for that group, which sounds complicated. Instead, could it be done by letting the browser store previous entries to the 'expert email address field' on the pop-up form for forwarding and offering them? For dragonflies we really only have a handful of such expert referees.

**Comment by DavidHepper** [comment]
I've had another request to reinstate this feature.
See also #560, which may just be a duplicate.

**Comment by DavidHepper** [comment]
Bump. In the latest case I'd like verifiers to be able to send larval records to an ID expert. We can live without a picklist of the names and email addresses of the experts.

**Comment by kitenetter** [comment]
@johnvanbreda how big a job would it be to reinstate the "email to an expert" function?

The workaround is to simply send a URL for the occurrence to someone else via normal email, but that provides a cut-down view of the record details and location unless the recipient is logged in to iRecord (and also would not provide any audit trail within iRecord to log the action taken)..

**Comment by johnvanbreda** [comment]
@kitenetter I'll have a look - probably a smallish task.

**Comment by johnvanbreda** [comment]
Now done.
